server.go

```diff
@@ -31,12 +33,26 @@ type quoteResponse struct {
 	cyrptokey string
 }
 
+var (
+	delayVal = rand.Intn(4) + 1
```

Comment:
Which is alright just for basic quicktests. Obviously you should use the regular random when fulltesting.